### PR TITLE
add helm chart

### PR DESCRIPTION
**Pull request by sfunkhouser**

### CODE REVIEW

File: chart/governor-api/templates/configmap.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+data:
+  .governor.yaml: |
+    admin-groups:
+      - {{ .Values.governor.adminGroups }}
+
+    nats:
+      {{- if .Values.nats.url }}
+      url: {{ .Values.nats.url }}
+      {{- else }}
+      url: tls://governor-nats.{{ .Values.clusterInfo.fqdn }}:4222
```
Comment (fishnix) [inline chart/governor-api/templates/configmap.yml:18]: do we need to remove this? (not sure if `clusterInfo` is something we expose - i didnt see it in the other oss org for any active repos)

File: chart/governor-api/templates/ingress.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+kind: Ingress
+metadata:
+  annotations:
+    cert-manager.io/cluster-issuer: letsencrypt-prod
+  name: {{ template "common.names.fullname" . }}-api
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+spec:
+  ingressClassName: nginx-governor
+  rules:
+    - host: {{ .Values.api.ingress.prefix }}.{{ .Values.clusterInfo.fqdn }}
```
Comment (fishnix) [inline chart/governor-api/templates/ingress.yml:12]: here too

File: chart/governor-api/templates/ingress.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+            backend:
+              service:
+                name: {{ template "common.names.fullname" $ }}-api
+                port:
+                  name: http
+    {{- end }}
+
+  tls:
+    - hosts:
+        - {{ .Values.api.ingress.prefix }}.{{ .Values.clusterInfo.fqdn }}
```
Comment (fishnix) [inline chart/governor-api/templates/ingress.yml:44]: and here

**Review by fishnix — DISMISSED** [review]
I only had the one questions about `clusterInfo` - its probably fine since it can be specified in the values....

File: chart/governor-api/templates/configmap.yml (hunk 1 above)
Comment (sfunkhouser) [reply chart/governor-api/templates/configmap.yml:18]: good catch, thanks!

**Review by fishnix — APPROVED** [review]
lgtm 👍

**Pull request merged**